dimod/roof_duality/fix_variables.py

```diff
@@ -0,0 +1,68 @@
+from dimod.roof_duality._fix_variables import fix_variables_wrapper
+from dimod.vartypes import Vartype
+
+
+def fix_variables(bqm, sampling_mode=True):
+    """Determine assignments for some bqm variables.
+
+    fix_variables uses maximum flow in the implication network to correctly fix variables (that is,
+    one can find an assignment for the other variables that attains the optimal value). The
+    variables that roof duality fixes will take the same values in all optimal solutions.
+
+    Args:
+        bqm (:obj:`.BinaryQuadraticModel`)
+            A binary quadratic model.
+
+        sampling_mode (bool, optional, default=True):
```

Comment:
sampling_mode description is unclear. What is sampling mode (i.e., what does it do for true vs false)?

Reply:
Agree, it's not clear you get multiple samples for the same ground state energy when `sampling_mode=True`.